app/src/main/java/net/gsantner/opoc/frontend/filebrowser/GsFileBrowserListAdapter.java

```diff
@@ -184,15 +184,14 @@ public void onBindViewHolder(@NonNull FilesystemViewerViewHolder holder, int pos
             }
         }
 
-        holder.description.setTextColor(ContextCompat.getColor(_context, _dopt.secondaryTextColor));
+        final boolean isFile = file.isFile();
 
-        holder.image.postDelayed(() -> {
```

Comment:
@gsantner You added this postDelayed in 2022. Do you remember why? 

The code works well without it...

Reply:
Very likely because it was crashing on some devices without it.